_resources/port1.0/group/golang-1.0.tcl

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+# This PortGroup accommodates golang projects hosted at GitHub.
+#
+# Usage:
+#
+# PortGroup     golang 1.0
+#
+# go.setup      author project 1.0.0 v # Same as github.setup
```

Comment:
This presumes the project is hosted on github?

Reply:
Yes. In principle of course golang projects could be hosted anywhere, but in practice I've never encountered one outside of GitHub. Some of the dependency-handling bits and path-mangling assume things are hosted on GitHub and without a concrete counterexample I thought it would be very difficult to handle other cases.

I am very much open to suggestions on improving this if anyone has any.

Reply:
Good point. I've changed it back to `github.setup`.

Reply:
The language in the comment still seems to be there though even though go.setup no longer exists. Can that be fixed?

Reply:
Yes, sorry, I had missed the comment. Now fixed.